### PR TITLE
Fix R0 mock test issue

### PR DESCRIPTION
**Pull request by Alancere**
compute,network,keyvault,resources,storage

### CODE REVIEW

File: specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-11-01-preview/examples/DeletedManagedHsm_Purge.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "api-version": "2021-11-01-preview"
   },
   "responses": {
-    "202": {}
+    "202": {},
```
Comment (tadelesh) [inline specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-11-01-preview/examples/DeletedManagedHsm_Purge.json:9]: There's no `200` response definition from swagger. Leave it to service team.

File: specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/examples/VirtualNetworkGatewayConnectionReset.json [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-    "api-version": "2020-08-01",
+    "api-version": "2021-05-01",
     "subscriptionId": "subid",
     "resourceGroupName": "rg1",
     "virtualNetworkGatewayConnectionName": "conn1"
   },
   "responses": {
-    "202": {}
+    "202": {},
+    "200": {}
```
Comment (tadelesh) [inline specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/examples/VirtualNetworkGatewayConnectionReset.json:10]: DITTO.

File: specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/examples/VpnSiteLinkConnectionReset.json [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
     "gatewayName": "gateway1",
     "connectionName": "vpnConnection1",
     "linkConnectionName": "Connection-Link1"
   },
   "responses": {
-    "202": {}
+    "202": {
+      "body": {}
+    },
+    "200": {
```
Comment (tadelesh) [inline specification/network/resource-manager/Microsoft.Network/stable/2021-05-01/examples/VpnSiteLinkConnectionReset.json:14]: DITTO.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/examples/FileShareAclsPatch.json [hunk 4]
```diff
@@ -4,15 +4,15 @@
...
+    "api-version": "2021-09-01",
     "monitor": "true",
     "fileShare": {
       "properties": {
         "signedIdentifiers": [
           {
             "id": "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTI",
             "accessPolicy": {
-              "startTime": "2021-04-01T08:49:37.0000000Z",
+              "startTime": "2021-09-01T08:49:37.0000000Z",
```
Comment (tadelesh) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/examples/FileShareAclsPatch.json:15]: Why this time need to change?

**Review by tadelesh — CHANGES_REQUESTED** [review]

**Review by heaths — APPROVED** [review]

**Review by heaths — COMMENTED** [review]
Key Vault changes LGTM, but looks like the service team wants to make the changes.

**Pull request closed without merging**